Let go decide when CGO is needed and when not.

--- a/build.go
+++ b/build.go
@@ -544,7 +544,7 @@ func setBuildEnvVars() {
 	os.Setenv("GOOS", goos)
 	os.Setenv("GOARCH", goarch)
 	os.Setenv("CC", cc)
-	if os.Getenv("CGO_ENABLED") == "" {
+	if false && os.Getenv("CGO_ENABLED") == "" {
 		switch goos {
 		case "darwin", "solaris":
 		default:
